server/src/main/java/org/opensearch/search/aggregations/metrics/SumIntegralAggregator.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/*
+ * Licensed to Elasticsearch under one or more contributor
```

Comment:
I think you can omit this header

Reply:
I do not know if this is something that I should omit as every other file in the server section of the code shares the same header, I think it might be some legal thing.